win32/src/file_win32.c

```diff
@@ -60,4 +41,472 @@ static void on_file_io_complete_win32(
     ULONG io_result,
     ULONG_PTR number_of_bytes_transferred,
     PTP_IO io
-);
\ No newline at end of file
+);
+
+static VOID NTAPI on_close_threadpool_group_member(
+     PVOID object_context, /*what was passed @  CreateThreadpoolIo*/
+     PVOID cleanup_context /*what was passed @  CloseThreadpoolCleanupGroupMembers*/
+)
+{
+    (void)object_context;
+    (void)cleanup_context;
+}
+
+IMPLEMENT_MOCKABLE_FUNCTION(
+    ,
+    FILE_HANDLE, file_create,
+    EXECUTION_ENGINE_HANDLE, execution_engine,
+    const char*, full_file_name,
+    FILE_REPORT_FAULT, user_report_fault_callback,
+    void*, user_report_fault_context
+)
+{
+    FILE_HANDLE handle = (FILE_HANDLE)NULL;
+    bool result = true;
+    if
+    (
+        /*Codes_SRS_FILE_43_033: [ If execution_engine is NULL, file_create shall fail and return NULL. ]*/
+        /*Codes_SRS_FILE_WIN32_43_040: [ If execution_engine is NULL, file_create shall fail and return NULL. ]*/
+        (execution_engine == NULL) ||
+        /*Codes_SRS_FILE_43_002: [ If full_file_name is NULL then file_create shall fail and return NULL. ]*/
+        /*Codes_SRS_FILE_WIN32_43_048: [ If full_file_name is NULL then file_create shall fail and return NULL. ]*/
+        (full_file_name == NULL) ||
+        /*Codes_SRS_FILE_43_037: [ If full_file_name is an empty string, file_create shall fail and return NULL. ]*/
+        /*Codes_SRS_FILE_WIN32_43_059: [ If full_file_name is an empty string, file_create shall fail and return NULL. ]*/
+        (strlen(full_file_name) == 0)
+    )
+    {
+        LogError("Invalid arguments to file_create: EXECUTION_ENGINE_HANDLE execution_engine=%p, const char* full_file_name=%s, FILE_REPORT_FAULT user_report_callback=%p, void* user_report_faul_context=%p",
+            execution_engine, full_file_name, user_report_fault_callback, user_report_fault_context);
+        result = false;

```

Comment:
>result = false; [](start = 8, length = 15)

E.g. here would say `result = NULL`, which matches with the requirement you have (shall return NULL).